third_party/accessibility/base/string_utils.cc

```diff
@@ -18,6 +20,48 @@
 
 namespace base {
 
+using double_conversion::DoubleToStringConverter;
+using double_conversion::StringBuilder;
+
+namespace {
+static char const kExponentChar = 'e';
+static const char* const kInfinitySymbol = "Infinity";
+static const char* const kNaNSymbol = "NaN";
+
+// The number of digits after the decimal we allow before switching to
+// exponential representation.
+constexpr int kDecimalInShortestLow = -6;
+// The number of digits before the decimal we allow before switching to
+// exponential representation.
+constexpr int kDecimalInShortestHigh = 12;
+constexpr int kConversionFlags =
+    DoubleToStringConverter::EMIT_POSITIVE_EXPONENT_SIGN;
+
+static const double_conversion::DoubleToStringConverter*
+GetDoubleToStringConverter() {
```

Comment:
No need to declare `static` since functions in an anonymous namespace are already scoped to the translation unit.

Reply:
This could return a const ref instead of a const ptr.

Reply:
I'm aware of the anonymous namespace properties, the static came from the Chrome code.  I'll clean that up.